lib/utils.js

```diff
@@ -13,16 +15,19 @@ const internals = {};
 
 module.exports.log = function (err, options, callback) {
 
-    const log = Fs.createWriteStream(options.logPath, options.writeStreamOptions);
-    const formattedErr = {
-        message: err.message,
-        stack: err.stack,
-        timestamp: Date.now()
-    };
+    Mkdirp(Path.dirname(options.logPath), () => {
```

Comment:
It seems like the best method to cover that scenario would be to mock the `Mkdirp` call and just make it always return an error for that test.